dashboards/node.libsonnet

```diff
@@ -127,14 +127,19 @@ local gauge = promgrafonnet.gauge;
           ],
         };
 
-      local diskSpaceUsage = graphPanel.new(
-        'Disk Space Usage',
-        datasource='$datasource',
-        span=6,
-        format='percentunit',
-      ).addTarget(prometheus.target(
-        'node:node_filesystem_usage:{%(clusterLabel)s="$cluster"}' % $._config, legendFormat='{{device}}',
```

Comment:
While I agree that this recording rule is missing the instance label to filter with, I don't think we should move away from using a recording rule here.
Either we adjust the current one or create a new one with an adjusted name.

Reply:
in here we simply need to reference the updated recording rule adding the `instance="$instance"` selector.